packages/autorest.python/test/azure/version-tolerant/Expected/AcceptanceTests/CustomPollerPagerVersionTolerant/custompollerpagerversiontolerant/aio/operations/_operations.py

```diff
@@ -289,6 +290,92 @@ async def get_next(next_link=None):
 
         return AsyncCustomPager(get_next, extract_data)
 
+    @distributed_trace
```

Comment:
we should still generate overloads for paging in this scenario (because we are automatically overloading the parameters type)

Reply:
resolved: https://github.com/Azure/autorest.python/pull/1407/commits/41c32a5419faa1055609e02f3f296727a3887813